dbt/deprecations.py

```diff
@@ -33,6 +33,16 @@ class SeedDropExistingDeprecation(DBTDeprecation):
   will be removed in a future version of dbt."""
 
 
+class OldStyleAdapterFunctionsDeprecations(DBTDeprecation):
+    # TODO add link to docs
+    name = 'old-style-adapter-functions'
+    description = """
+    This is an old-style adapter function (takes a schema and identifier
+    instead of a Relation object). Please use the new-style functions.
+    You can find more information in the documentation. (ADD LINK)
```

Comment:
I can help with the docs here

Reply:
Additionally, it would be great if we could interpolate the name of the function being called and the model name (if possible). Otherwise, I think it might be unclear which function is deprecated

Reply:
yeah, that's a good idea. will do